Deploy/stacks/dynamic/stack-clients/src/main/java/com/cmclinnovations/stack/clients/docker/DockerClient.java

```diff
@@ -291,7 +299,7 @@ public boolean fileExists(String containerId, String filePath) {
     }
 
     public void makeDir(String containerId, String directoryPath) {
-        executeSimpleCommand(containerId, "mkdir", "-p", directoryPath);
+        createComplexCommand(containerId, "mkdir", "-p", directoryPath).withUser("root").exec();
```

Comment:
The ```user``` should probably be optional as always setting it to ```root``` might stop containers that don't run as root from being able to access the files.

Reply:
resolved in later commits